src/plugins/ui_core_views/custom_colors.ts

```diff
@@ -107,7 +109,8 @@ export class CustomColorsPlugin extends UIPlugin {
     ]).filter((color) => !COLOR_PICKER_DEFAULTS.includes(color));
   }
 
-  private getColorsFromCells(cells: Cell[]): Color[] {
+  private getColorsFromCells(sheetId: UID): Color[] {
+    const cells: Cell[] = Object.values(this.getters.getCells(sheetId));
```

Comment:
```suggestion
    const cells = Object.values(this.getters.getCells(sheetId));
```
Type is inferred, no need to add it